apix_documentation_swagger_v4.yaml

```diff
@@ -0,0 +1,1186 @@
+              type: string
+              description: ""
+            signup_url:
+              type: string
+              description: ""
+            password_url:
+              type: string
+              description: ""
+            color:
+              type: string
+              description: ""
+            logo:
+              type: string
+              description: ""
+            created_at:
+              type: string
+              format: date-time
+              description: ""
+            updated_at:
+              type: string
+              format: date-time
+              description: ""
+
+    MetaData:
+      type: object
+      required:
+        - meta
+      properties:
+        meta:
+          type: object
+          required:
+            - total
+            - total-pages
+          properties:
+            total:
+              type: integer
+            total-pages:
+              type: integer
+
+    NavigationLinks:
+      type: object
+      required:
+        - links
+      properties:
+        links:
+          type: object
+          required:
+            - self
+          properties:
+            self:
+              type: string
+            next:
+              type: string
+            prev:
+              type: string
+
+    Migration:
+      allOf:
+        - $ref: '#/components/schemas/GenericData'
+      type: object
+      required:
+        - attributes
+      properties:
+        attributes:
+          type: object
+          required:
+            - status
+            - requested_at
+          properties:
+            status:
+              description: Migration's status
+              type: string
+            requested_at:
+              description: Timestamp of when the migration was requested
+              type: string
+            migrated_at:
+              description: Timestamp of when the migration succeeded
+              type: string
+            error:
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/Error_Info'
+
+    Basic_Credentials:
+      description: Includes the basic information to access an end user's
+        account
+      type: object
+      properties:
+        login:
+          type: string
+          description: The account's username/email address/identifier
+          example: "jdoe@successful.ii"
+        password:
+          type: string
+          description: The account's password
+          example: "HeLl0o1!"
+      required:
+        - login
+        - password
+
+    Error_Info:
+      type: object
+      required:
+        - code
+        - detail
+      properties:
+        code:
+          description: Error code
+          type: string
+        detail:
+          description: Description of the error encountered
+          type: string
+        source:
+          type: object
+          required:
+            - parameter
+          properties:
+            parameter:
+              type: string
```

Comment:
Should we document the example of the errors that they can get?

Reply:
I think it makes more sense to add examples in the endpoints themselves. I'll double-check whether some use cases for several examples are already supported in Swagger-UI (see https://arcusfi.atlassian.net/browse/USX-272); a few days ago it was mentioned that a feature for these would soon be released.